### PR TITLE
Add ability to guess raw language type using content-type headers

### PR DESCRIPTION
**Pull request by rutvik-gs**
Adding support in `curlConverter` to guess raw dataMode language type using headers if available

### CODE REVIEW

File: src/lib.js [hunk 1]
```diff
@@ -838,6 +849,14 @@ var program,
             else {
               request.body.mode = 'raw';
               request.body.raw = rawDataString;
+
+              request.body.options = {};
+              request.body.options.raw = {};
+              request.body.options.raw.language = Object.keys(LANGUAGE_REGEX_MATCH)
+                .find((key) => {
+                  return LANGUAGE_REGEX_MATCH[key].test(content_type);
```
Comment (VShingala) [inline src/lib.js:857]: Can we wrap this around try catch? This is to make sure regex doesn't break entire import in case of bad input.

**Review by VShingala — COMMENTED** [review]
Added one comment, rest looks good.

File: src/lib.js (hunk 1 above)
Comment (rutvik-gs) [reply src/lib.js]: Makes sense. I have made the change.

**Review by VShingala — APPROVED** [review]
PR looks good. I'll merge this and create a release.

**Pull request merged**